universal_transfer_operator/docs/development/RELEASE.md

```diff
@@ -0,0 +1,82 @@
+# Managing Releases
+
+## Updating the changelog
+
+The first step to creating a release is identifying what changed from the previous release and adding a description to the [CHANGELOG.md](../CHANGELOG.md).
+
+One way to identify what changed from the previous release is to use a comparison URL between the previous release branch and the main branch.
+
+Example -
+[https://github.com/astronomer/astro-sdk/compare/1.3.3...main](https://github.com/astronomer/astro-sdk/compare/1.3.3...main)
+
+This will show all the commits that are in the main branch but not in the release branch. Using this we can note all the changes significant enough to be mentioned in CHANGELOGS.
+
+
+## Handling patch releases
+
+### When should I create a patch release?
+Patch releases are exclusively for bugs and security fixes. Patch releases can have minor _enhancements_ to
+existing features but really should not affect the API in any noticeable way (unless it is adding an argument). For any
+functionality changes or new endpoints, please create a minor or major release.
+
+A patch release should increment the third digit e.g. `0.6.9 -> 0.6.10`. This indicates to users that the release is
+strictly for bug fixes, and they can easily upgrade to it.
+
+
+### Creating a patch release
+
+When looking through the branches on the main repo, you will notice a series of `release` branches labeled
+`release-<major version>.<minor version>`. So if we are currently working on the 0.6 release, the branch should be labeled
+`release-0.6`. Before creating a minor release please check the [milestones](https://github.com/astronomer/astro-sdk/milestones)
+page to ensure that all relevant bug fixes have been PRed and merged.
+
+Once you merge all expected fixes, the next step is to ensure that all fixes have been cherry-picked to the release
+branch from main (please use `git cherry-pick -x <commit id>` to retain
+git commit hashes and messages). Depending on the fix there might be some git conflicts to resolve. If you run into conflicts, please
+resolve said conflicts, run `git add .`, and then `git cherry-pick --continue` to continue the merge.
+
+After cherry-picking all needed fixes, follow the instructions to [create a release from the release branch](https://github.com/astronomer/astro-sdk/blob/main/python-sdk/docs/development/RELEASE.md#creating-a-release-from-the-release-branch).
```

Comment:
@sunank200 Path needs to be updated.

Reply:
[Done](https://github.com/astronomer/astro-sdk/pull/1861/commits/eb17c53b7d7e836fe1dffa8d93f329bfe06190ef)